### PR TITLE
MSAL-Node NetworkModule: Fixed inconsistencies with cancellationToken (timeout)

### PR DESCRIPTION
**Pull request by Robbie-Microsoft**
- renamed cancellationToken to timeout
- it can only be used in get requests (region discovery, for example)
- it's not part of the public api

### CODE REVIEW

**Review by bgavrilMS — APPROVED** [review]

File: lib/msal-common/src/utils/Constants.ts [hunk 1]
```diff
@@ -64,22 +64,26 @@ export const Constants = {
 };
 
 export const HttpStatus = {
+    SUCCESS: 200,
```
Comment (neha-bhargava) [inline lib/msal-common/src/utils/Constants.ts:67]: This already exists below at line 69

File: lib/msal-common/src/utils/Constants.ts [hunk 2]
```diff
@@ -64,22 +64,26 @@ export const Constants = {
...
     REDIRECT: 302,
+    CLIENT_ERROR: 400,
     CLIENT_ERROR_RANGE_START: 400,
     BAD_REQUEST: 400,
     UNAUTHORIZED: 401,
     NOT_FOUND: 404,
     REQUEST_TIMEOUT: 408,
     TOO_MANY_REQUESTS: 429,
     CLIENT_ERROR_RANGE_END: 499,
+    SERVER_ERROR: 500,
```
Comment (neha-bhargava) [inline lib/msal-common/src/utils/Constants.ts:79]: Duplicate of INTERNAL_SERVER_ERROR at line 82

**Review by neha-bhargava — CHANGES_REQUESTED** [review]
Since the managed identity calls use GET request, please evaluate how this timeout affects MSI calls.

File: lib/msal-common/src/network/INetworkModule.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ export interface INetworkModule {
     sendGetRequestAsync<T>(
         url: string,
         options?: NetworkRequestOptions,
-        cancellationToken?: number
+        timeout?: number
```
Comment (bgavrilMS) [inline lib/msal-common/src/network/INetworkModule.ts:34]: Can you please add a comment on the public API that is deprecated? 

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/23df98f6747f47f9ba1bb458feba9f183f4a902c/lib/msal-common/src/error/AuthError.ts#L18

Comment (Robbie-Microsoft) [reply lib/msal-common/src/network/INetworkModule.ts:34]: This isn't necessary. This is only changing the parameter name, and cancellationToken/timeout is only used internally.

**Review by neha-bhargava — APPROVED** [review]

**Pull request merged**